README.md

```diff
@@ -21,7 +21,6 @@
 
 <p>
 	<a href="https://github.com/mindsdb/mindsdb/actions"><img src="https://github.com/mindsdb/mindsdb/workflows/MindsDB%20workflow/badge.svg" alt="MindsDB workflow"></a>
-	<a href="https://www.python.org/downloads/" target="_blank"><img src="https://img.shields.io/badge/python-3.7.x%20|%203.8.x|%203.9.x-brightgreen.svg" alt="Python supported"></a>
```

Comment:
Please do not remove this line. We want exclude just 3.7.x version and keep the other versions.

Please replace `img src` value with this one: https://img.shields.io/badge/python-3.8.x%7C%203.9.x-brightgreen.svg